include/client/tickets.inc.php

```diff
@@ -205,19 +205,19 @@
     <thead>
         <tr>
             <th nowrap>
-                <a href="tickets.php?sort=ID&order=<?php echo $negorder; ?><?php echo $qstr; ?>" title="Sort By Ticket ID"><?php echo __('Ticket #');?>&nbsp;<i class="icon-sort"></i></a>
+                <a href="tickets.php?sort=ID&order=<?php echo $negorder; ?><?php echo $qstr; ?>" title="<?php echo __('Sort By Ticket ID');?>"><?php echo __('Ticket #');?>&nbsp;<i class="icon-sort"></i></a>
```

Comment:
Developer internal note: We need to use `sprintf` instead to break apart "sort by" and the sort into separate translatable strings.